src/ThreeEditor/components/Editor/EditorAppBar/components/EditorTitlebar.tsx

```diff
@@ -21,7 +21,6 @@ export function EditorTitleBar() {
 	const [editMode, setEditMode] = useState(false);
 	const [title, setTitle] = useState<string>(yaptideEditor?.config.getKey('project/title'));
 	const titleRef = useRef(title);
-	useDocumentTitle(titleRef.current);
```

Comment:
`useDocumentTitle` is still imported even though its only purpose was removed. Clear imports and remove [usehooks-ts](https://www.npmjs.com/package/usehooks-ts) if no other features require it.